README.md

```diff
@@ -108,6 +108,10 @@ Dont forget to add your user into dialout group "sudo usermod -aG dialout YOUR_U
 
 If you have 3D model animation problems, enable "Override software rendering list" in Chrome flags chrome://flags/#ignore-gpu-blacklist
 
+### Windows users
+
+If you experience smudged/dithered fonts issues, try invoking the .exe executable file with the --disable-gpu switch. This will switch off hardware graphics acceleration. Likewise, setting your graphics card setting FXAA (Antialiasing) to OFF might be a remedy as well.
```

Comment:
I think 'FXAA' is proprietary to one GPU manufacturer (Nvidia?), so this should probably be mentioned.